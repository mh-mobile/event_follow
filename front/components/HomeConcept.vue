<template lang="pug">
.home_concept
  .logo
    img(src="@/assets/logo_transparent.png")
  div
    | 興味のある技術
  div
    | イベントを見逃さない
</template>

<style lang="scss" scoped>
@media all and (min-width: 480px) {
  .home_concept {
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    font-size: 46px;
    font-weight: bold;
    line-height: 58px;
    margin-bottom: 20px;
    color: #52648d;

    .logo {
      padding-top: 100px;
      padding-bottom: 100px;
      width: 250px;
      height: auto;
    }
  }
}

@media all and (max-width: 480px) {
  .home_concept {
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    font-size: 1.8em;
    font-weight: bold;
    margin-bottom: 20px;
    color: #52648d;
    line-height: 40px;

    .logo {
      padding-top: 50px;
      padding-bottom: 30px;
      width: 150px;
      height: auto;
    } 
  }
}
</style>
