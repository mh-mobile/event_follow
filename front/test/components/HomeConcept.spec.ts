import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import HomeConcept from "@/components/HomeConcept.vue"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
describe("HomeConcept.vue", () => {
  it("renders html correctly", () => {
    const wrapper = shallowMount(HomeConcept, {
      localVue
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
