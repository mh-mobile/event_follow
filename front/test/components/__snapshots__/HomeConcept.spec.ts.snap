// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeConcept.vue renders html correctly 1`] = `
"<div class=\\"home_concept\\">自分の興味の近い技術系の<br>
  イベントを発見するためのソーシャルサービス</div>"
`;
